doc/whats_new.rst

```diff
@@ -7,312 +7,140 @@
 What's new?
 ===========
 
-Here we list a changelog of MNE-BIDS.
+.. currentmodule:: mne_bids
+
+.. _changes_0_5:
+
+Version 0.5 (2020-10-xx)
+------------------------
+
+This is a **big** release with lots of changes, many of them breaking existing
+code. But do not fear: migration is easy, and you will **love** what we have
+been cooking for you!
 
 .. contents:: Contents
    :local:
    :depth: 3
 
-.. currentmodule:: mne_bids
+Notable changes
+~~~~~~~~~~~~~~~
+- We introduce `mne_bids.BIDSPath`, a new class for all BIDS file and folder
+  operations. All functions in MNE-BIDS that previously accepted filenames
+  and folder locations (e.g. ``bids_root``) have been updated to work with
+  ``BIDSPath``. Others have been removed.
+  Consequently, you will need to update your existing code, too.
+  See the `API changes`_ section for an overview of which functions have
+  changed or have been removed, and follow
+  :ref:`this introduction<bidspath-intro>`
+  to learn about the basics of ``BIDSPath``. Don't worry – it's going to be a
+  breeze! 🤗
+
+- MNE-BIDS now requires MNE-Python 0.21.
+
+- The new function :func:`mne_bids.make_report` can help you populate a
+  paragraph of your next paper's methods section!
 
-.. _current:
+- You can now interactively mark channels as bad using
+  :func:`mne_bids.mark_bad_channels`.
+
+- Elekta/Neuromag/MEGIN fine-calibration and crosstalk files can now be stored
+  in your BIDS dataset via :func:`mne_bids.write_meg_calibration` and
+  :func:`mne_bids.write_meg_crosstalk`.
+
+- When writing a raw file that contains annotations, these will now be
+  converted to and stored as events by :func:`mne_bids.write_raw_bids`.
+
+
+Authors
+~~~~~~~
+The following people have contributed to this release of MNE-BIDS:
 
-Current
--------
+- `Adam Li`_
+- `Alexandre Gramfort`_
+- `Alex Rockhill`_
+- `Mainak Jas`_
```

Comment:
thanks for adding me :)

I see a name missing: Evgenii Kalenkovich

In MNE, we typically look at the commits. You can find a command in the release documentation. That way you don't miss out anyone.

Reply:
Austin Hurst (#456) maybe also missing

Reply:
```
 git shortlog v0.4..HEAD --summary --numbered --email
    41  Richard Höchenberger <richard.hoechenberger@gmail.com>
    35  Adam Li <adam2392@gmail.com>
    16  Alexandre Gramfort <alexandre.gramfort@m4x.org>
    13  Stefan Appelhoff <stefan.appelhoff@mailbox.org>
    10  kalenkovich <e.kalenkovich@gmail.com>
     6  Austin Hurst <mynameisaustinhurst@gmail.com>
     4  Alex Rockhill <aprockhill@mailbox.org>
     2  Alex Rockhill <aprockhill206@gmail.com>
     1  Robert Luke <748691+rob-luke@users.noreply.github.com>
```

we also need to update [.mailmap](https://github.com/mne-tools/mne-bids/blob/master/.mailmap) to map yet another email of Alex R :rofl:  --> and to maybe get the true name of kalenkovich and properly map it.

Reply:
Resolved in 4700e1a

Reply:
lgtm, thanks